content/en/developers/_index.md

```diff
@@ -33,6 +33,10 @@ If the solution you require is truly unavailable, you can contact [Datadog Suppo
 
 ### Partners and the Datadog Marketplace
 
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">Datadog Marketplace is not available for your selected <a href="/getting_started/site">Datadog site</a> ({{< region-param key="dd_site_name" >}}).</div>
```

Comment:
The Datadog Marketplace is now available in all sites (including Fed). Is this still needed?

Reply:
gotcha! removing marketplace fed messaging here